lib/commands/cookies.js

```diff
@@ -11,20 +12,22 @@ commands.getCookies = async function getCookies () {
 
   // get the cookies from the remote debugger, or an empty object
   const cookies = await this.remote.getCookies() || {cookies: []};
-  // the value is URI encoded, so decode it
-  // but keep all the rest of the info intact
-  let decodedCookies = [];
-  cookies.cookies.forEach(function mapCookie (cookie) {
+
+  // the value is URI encoded, so decode it safely
+  const decodedCookieValues = cookies.cookies.map(function mapCookieValue (cookie) {
```

Comment:
is there any particular reason to not use anonymous function here?

Reply:
I don't think so. I was only replicating the style of the original. Happy to change.

Reply:
I personally find it cleaner to use lambas unless we really need to use syntactical `this`

Reply:
Yeah, totally agree. 👍